src/tests/functional/plugin/conformance/test_runner/conformance_infra/include/conformance.hpp

```diff
@@ -44,19 +46,49 @@ inline ov::AnyMap readPluginConfig(const std::string &configFilePath) {
     return config;
 }
 
-inline std::vector<std::string> getModelPaths(const std::vector<std::string>& conformance_ir_paths) {
+inline std::vector<std::string> getModelPaths(const std::vector<std::string>& conformance_ir_paths,
+                                              const std::string opName = "") {
+    static std::list<std::string> dirList;
```

Comment:
to separate to global conformance variable `dirList`?
and looks like: listPrepared == !dirList.empty()

Reply:
I use it to prevent rescan folders each call in case of nothing found. For example, user could point on some huge folder, which doesn't have applicable *.xml files. But without additional flag - we will rerun folder scan each call of getModelPaths.
Using test case separation this method is calling for each operation, not once per-run.

Reply:
move to global - I'm ok.